sdk.nix

```diff
@@ -40,7 +40,7 @@ rec {
     packageOverrides = pkgs: rec {
       inherit (ugpkgs) docker go imagemagick linux nix;
       mysql = ugpkgs.mariadb;
-      php = ugpkgs.php54;
+      php = ugpkgs.php53;
```

Comment:
better add `php53` verbatim to the sdk: https://github.com/zalora/microgram/blob/master/sdk.nix#L138